OracleWebCenterSites/dockerfiles/12.2.1.4/sites-container-scripts/startAdminServer.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# Description: This script is used to start a WebLogic Admin server. And tail the logs.
+#
+
+export DOMAIN_NAME=$DOMAIN_NAME
+export DOMAIN_ROOT="/u01/oracle/user_projects/domains"
+export DOMAIN_HOME="${DOMAIN_ROOT}/${DOMAIN_NAME}"
+
+# Start Admin server
+cd $DOMAIN_HOME/bin
+
+echo ""
+echo "tail $DOMAIN_HOME/bin/weblogic.out in a new window"
+./startWebLogic.sh > weblogic.out 2>&1 &
+
+echo "waiting for server to start"
+
+counter=0
+while true; do
+    grep "RUNNING" $DOMAIN_HOME/bin/weblogic.out
```

Comment:
This is a seriously inefficient way to test if a server is up. Isn't there a more elegant way besides reading a file over and over?